src/gui/dialogs/lua_interpreter.cpp

```diff
@@ -321,7 +321,7 @@ class tlua_interpreter::input_model {
 #ifdef HAVE_HISTORY
 		// Do history expansions
 		char * cmd_cstr = new char [cmd.length()+1];
-		std::strcpy (cmd_cstr, cmd.c_str());
```

Comment:
This was a fix, because clang refused to compile this on my system. 
std::strcpy is contained in <cstring>, which is not included. However strcpy from "string.h" is included here somehow.